src/main/scala/org/hammerlab/guacamole/reads/Read.scala

```diff
@@ -62,8 +62,9 @@ trait Read {
 
   /** Is this read mapped? */
   final val isMapped: Boolean = this match {
-    case r: MappedRead   => true
-    case r: UnmappedRead => false
+    case r: MappedRead       => true
```

Comment:
No, not really, I think that makes sense as well. I think we did have that at some point, not sure what was the reasoning behind this way.

 Will follow on with that